packages/playwright-ct-vue2/hooks.d.ts

```diff
@@ -14,14 +14,20 @@
  * limitations under the License.
  */
 
-import { CombinedVueInstance, Vue } from 'vue/types/vue';
-
+import { ComponentOptions } from 'vue';
+import { CombinedVueInstance, Vue, VueConstructor } from 'vue/types/vue';
+ 
 type JsonPrimitive = string | number | boolean | null;
 type JsonValue = JsonPrimitive | JsonObject | JsonArray;
 type JsonArray = JsonValue[];
 type JsonObject = { [Key in string]?: JsonValue };
+type VueOptions = ComponentOptions<Vue> & { router?: unknown, store?: unknown, [key: string]: unknown };
```

Comment:
Why do we special-case `router?: unknown, store?: unknown, [key: string]: unknown` here?

Reply:
Ah, i think i forgot to delete that type. BTW i can either return the `VueOptions` or update it directly:

```tsx
beforeMount(async ({ Vue, options, hooksConfig }) => {
  Vue.use(Router);
  options.router = router;
});
```

vs

```tsx
beforeMount(async ({ Vue, hooksConfig }) => {
  Vue.use(Router);
  return { router }
});
```

I think i have a slight preference for returning `VueOptions` as updating props directly feels a kind of odd. Also the return is a bit more in line with:
```tsx
beforeMount(async ({ App }) => {
  return <Router><App /></Router>
});
```

Which one do you prefer?